types/2020-03-02/Issuing/Disputes.d.ts

```diff
@@ -18,24 +18,24 @@ declare module 'stripe' {
         /**
          * Disputed amount. Usually the amount of the `disputed_transaction`, but can differ (usually because of currency fluctuation or because only part of the order is disputed).
          */
-        amount: number;
+        amount?: number;
```

Comment:
Why did all dispute fields except `id` and `object` become optional?

Reply:
Internal reasons mostly